lib/svgo/coa.js

```diff
@@ -295,9 +304,11 @@ function optimizeFromString(svgstr, config, datauri, input, output) {
                 output = input;
             }
 
-            console.log('\r');
+            if (! config.quiet) {
```

Comment:
There are several places and this is only one of those. You have updated in some other place.